components/NavigationBar.js

```diff
@@ -22,8 +24,37 @@ export default function NavigationBar () {
 function NavigationLink ({ href, disabled, children }) {
   return (
     <span className={disabled ? 'disabled' : ''}>
-      <LevitatingLink href={href}>{children}</LevitatingLink>
-      <style jsx>{`
+      <Link to={href} spy smooth duration={500}>
```

Comment:
Using a `Link` component from `react-scroll` means that the user cannot Cmd+Click to open the link in a new tab.